File: lib/_satis/satis_request_model.dart
import 'dart:convert';

import 'package:json_annotation/json_annotation.dart';

import '../core/base/base_model.dart';

part 'satis_request_model.g.dart';

@JsonSerializable()
class SatisRequestModel extends BaseModel {
  String? barkod;
  int? subeKodu;
  int? kullanici;
  DateTime? tarih;
  DateTime? basTarih;
  DateTime? bitTarih;

  SatisRequestModel({
    this.barkod,
    this.subeKodu,
    this.kullanici,
    this.tarih,
    this.basTarih,
    this.bitTarih,
  });

  Map<String, dynamic> toJson() => _$SatisRequestModelToJson(this);
  factory SatisRequestModel.fromJson(Map<String, dynamic> json) => 
   _$SatisRequestModelFromJson(json);
  @override
  String myJson()  => jsonEncode(toJson());
}

File: GiroDashboard.Application/DTOs/Satis/SatisRequestModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GiroDashboard.Application.DTOs.Satis
{
    public class SatisRequestModel
    {
        public int? Kullanici { get; set; }
        public int? SubeKodu { get; set; }
        public DateTime? Tarih { get; set; }
        public DateTime? BasTarih { get; set; }
        public DateTime? BitTarih { get; set; }
        public string? Barkod { get; set; }
    }
}
